nimare/tests/test_diagnostics.py

```diff
@@ -60,8 +64,11 @@ def test_jackknife_with_zero_clusters(testdata_cbma_full):
     res = meta.fit(testdata_cbma_full)
 
     jackknife = diagnostics.Jackknife(target_image="z", voxel_thresh=10)
-    contribution_table, clusters_table, label_maps = jackknife.transform(res)
+    results = jackknife.transform(res)
 
+    contribution_table = results.tables["z_diag-Jackknife_counts"]
+    clusters_table = results.tables["z_clust"]
+    label_maps = results.maps["z_PositiveTail"]
```

Comment:
maybe a little unintuitive what PositiveTail means in this context, may need to make it more explicit that this is a label map and not a statistical map.

Reply:
and is the label map represented as a vector or a 3D array? I think the other maps are represented as 1D arrays.

Reply:
> maybe a little unintuitive what PositiveTail means in this context, may need to make it more explicit that this is a label map and not a statistical map.

Agreed. I've added a couple of new modalities to our bids-like format. See here: https://nimare--804.org.readthedocs.build/en/804/outputs.html#file-names

> and is the label map represented as a vector or a 3D array? I think the other maps are represented as vectors.

Good catch! I was missing the np.squeeze.

Thanks!